playbooks/docker-machine-functional-devstack/run.yaml

```diff
@@ -1,10 +1,10 @@
 - hosts: all
   become: yes
   roles:
+    - config-docker-machine
     - clone-devstack-gate-to-workspace
     - create-devstack-local-conf
     - install-devstack
-    - config-docker-machine
```

Comment:
Just want to make sure, does it pass in allinone? Others looks good.

Reply:
yes, you can see: http://80.158.17.129/builds?uuid=f62c7423653d41a2a9cff9777cb4ce3a